src/utils/registration-helpers.ts

```diff
@@ -760,31 +761,46 @@ export const registrationLanguageOptions = [
   getLanguageByIso6393Code('mis'),
 ];
 
+const isRegistration = (reg: Registration | RegistrationSearchItem2): reg is Registration => {
+  return (reg as Registration).entityDescription !== undefined;
```

Comment:
Kan virke som fromålet her er å se om dette er en full Publication eller et søketreff. En full Publication kan også mangle `entityDescription` (Start ny tom registrering feks), så da stemmer ikke denne sjekken. Typen ligger i, nettopp, typen :D 
```suggestion
  return reg.type === 'Publication';
```